### ISSUE TITLE
Globaleaks doesn't boot due to apparmor denying lsb_release exec

### ISSUE THREAD

**Issue by vincib** [globaleaks/GlobaLeaks#2747]
**Current behavior**

on latest bionic ubuntu (2019-12 upgraded) & latest globaleals (3.11.46) /etc/apparmor.d/usr.bin.globaleaks does not contains 

**/usr/bin/lsb_release x,**

which triggers the following error : 

`Dec  9 16:50:48 glpplaaf kernel: [  282.533581] audit: type=1400 audit(1575910248.601:29): apparmor="DENIED" operation="exec" profile="usr.bin.globaleaks" name="/usr/bin/lsb_release" pid=29608 comm="sh" requested_mask="x" denied_mask="x" fsuid=0 ouid=0
`
and globaleaks crash as follow : 

```
Dec  9 16:50:48 glpplaaf globaleaks[29537]: Traceback (most recent call last):
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/bin/globaleaks", line 9, in <module>
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     from distro import linux_distribution
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/distro.py", line 1051, in <module>
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     _distro = LinuxDistribution()
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/distro.py", line 595, in __init__
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     if include_lsb else {}
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/distro.py", line 932, in _get_lsb_release_info
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     raise subprocess.CalledProcessError(code, cmd, stdout, stderr)
Dec  9 16:50:48 glpplaaf globaleaks[29537]: subprocess.CalledProcessError: Command 'lsb_release -a' returned non-zero exit status 126.
Dec  9 16:50:48 glpplaaf globaleaks[29537]: Error in sys.excepthook:
Dec  9 16:50:48 glpplaaf globaleaks[29537]: Traceback (most recent call last):
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 145, in apport_excepthook
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     os.O_WRONLY | os.O_CREAT | os.O_EXCL, 0o640), 'wb') as f:
Dec  9 16:50:48 glpplaaf globaleaks[29537]: PermissionError: [Errno 13] Permission denied: '/var/crash/_usr_bin_globaleaks.0.crash'
Dec  9 16:50:48 glpplaaf globaleaks[29537]: Original exception was:
Dec  9 16:50:48 glpplaaf globaleaks[29537]: Traceback (most recent call last):
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/bin/globaleaks", line 9, in <module>
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     from distro import linux_distribution
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/distro.py", line 1051, in <module>
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     _distro = LinuxDistribution()
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/distro.py", line 595, in __init__
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     if include_lsb else {}
Dec  9 16:50:48 glpplaaf globaleaks[29537]:   File "/usr/lib/python3/dist-packages/distro.py", line 932, in _get_lsb_release_info
Dec  9 16:50:48 glpplaaf globaleaks[29537]:     raise subprocess.CalledProcessError(code, cmd, stdout, stderr)
Dec  9 16:50:48 glpplaaf globaleaks[29537]: subprocess.CalledProcessError: Command 'lsb_release -a' returned non-zero exit status 126.
Dec  9 16:50:48 glpplaaf globaleaks[29537]:    ...fail!

```

**Expected behavior**

globaleaks starts at runtime

**Steps to reproduce the problem or feature illustration**

* latest bionic (updates 2019-12-09)
* globaleaks installed a few month ago, and upgraded too

**GlobaLeaks version:**

3.11.46

**Server Operating System and Version (if applicable):**

bionic

**Comment by evilaliv3** [comment]
Thank you @vincib 

We confirm this issue and we are already working on a patch to be issued by today.

**Comment by evilaliv3** [comment]
@vincib: This has been fixed starting from release 3.11.48

Thank you for reporting the failure with your system.
